integration-tests/zendeskRequestAPI.spec.ts

```diff
@@ -1,12 +1,85 @@
 import axios from 'axios'
-import { getEnvVariable } from './lib/zendeskParameters'
+import crypto from 'crypto'
+
+//TODO: test setup and teardown for creating and deleting a ticket to use
+
+const baseUrl = process.env.ZENDESK_WEBHOOK_API_BASE_URL as string
```

Comment:
If you merge in main, there's already an env variable 'AWS_WEBHOOK_URL' that holds this.

Reply:
I changed it on purpose because we didn't understand what it was at first. I think this makes it a bit clearer?

Reply:
Also we will be adding more API gateways so we need to make a distinction between them